src/components/Tabs/Tabs.tsx

```diff
@@ -35,7 +38,7 @@ export interface State {
   tabToFocus: number;
 }
 
-export default class Tabs extends React.PureComponent<Props, State> {
+export class Tabs extends React.PureComponent<CombinedProps, State> {
```

Comment:
We had to export classes in a previous TS version however I don't think that's the case anymore. We should be able to 🔥 it
```suggestion
class Tabs extends React.PureComponent<CombinedProps, State> {
```